core/src/main/kotlin/model/Documentable.kt

```diff
@@ -364,7 +365,7 @@ data class DTypeAlias(
 
 sealed class Projection
 sealed class Bound : Projection()
-data class OtherParameter(val name: String) : Bound()
+data class OtherParameter(val dri: DRI, val declarationDRI: DRI, val name: String) : Bound()
```

Comment:
I'm not sure why we need both DRIs here.

Reply:
Changed to having only the declaration